lib/engine/game/g_18_ardennes/market.rb

```diff
@@ -16,7 +16,7 @@ module Market
         ).freeze
         MARKET_TEXT = Base::MARKET_TEXT.merge(
           par_1: 'Minor company starting values',
```

Comment:
Is "Minor company" correct?

Reply:
I don't believe Toby established a protocol for minors.